dotcom-rendering/src/components/LiveBlogBlocksAndAdverts.tsx

```diff
@@ -59,27 +59,49 @@ export const LiveBlogBlocksAndAdverts = ({
 		);
 	}
 
-	let numPixelsOfContentWithoutAdvert = 0;
-	let numAdvertsInserted = 0;
+	let pxSinceAdMobile = 0;
+	let mobileAdCounter = 0;
+
+	let pxSinceAdDesktop = 0;
+	let desktopAdCounter = 0;
 
 	return (
 		<>
 			{blocks.map((block, i) => {
-				numPixelsOfContentWithoutAdvert +=
-					calculateApproximateBlockHeight(block.elements);
-
-				const willDisplayAdAfterBlock =
+				pxSinceAdMobile += calculateApproximateBlockHeight(
+					block.elements,
+					true,
+				);
+				const willinsertAdMobile =
```

Comment:
Very picky but could we keep consistent camelCase for this var name? `willinsertAdMobile` -> `willInsertAdMobile`

Reply:
Absolutely! Not picky at all